Text Generation WebUI/src/configuration-parser.ts

```diff
@@ -104,7 +104,7 @@ export class ConfigurationParser {
     this.cliParams.push(`--loader`, modelLoader);
 
     const loaderConfiguration = modelLoaderSettings[
-      `${modelLoader.toLowerCase()}_options` as keyof typeof modelLoaderSettings
+      `${modelLoader.toLowerCase().replaceAll('.', '')}_options` as keyof typeof modelLoaderSettings
```

Comment:
Do this change in `@super-protocol/solution-utils` please - https://github.com/Super-Protocol/solution-utils/blob/main/src/solution-configuration/get-cli-params.ts#L118